.evergreen/download-mongodb.sh

```diff
@@ -504,3 +505,31 @@ download_and_extract ()
    ./mongodb/bin/mongod --version
    cd -
 }
+
+# download_and_extract downloads a requested MongoDB server package.
+# If the legacy shell is not included in the download, the legacy shell is also downloaded from the 6.0 package.
+download_and_extract ()
+{
+   MONGODB_DOWNLOAD_URL=$1
+   EXTRACT=$2
+
+   download_and_extract_package "$MONGODB_DOWNLOAD_URL" "$EXTRACT"
+
+   if [ ! -e $DRIVERS_TOOLS/mongodb/bin/mongo ]; then
```

Comment:
Does this need to work on Windows, wherein the file will have a `.exe` suffix?

Reply:
Great question. Yes. This has been updated. Curiously the `cp` command did not error on the [Windows test run](https://evergreen.mongodb.com/lobster/evergreen/task/mongo_c_driver_windows_2017_test_latest_server_auth_sspi_winssl_patch_1841a0eb0d344abf7bd84413807b1e15e3c92496_6280e09830661571add434b5_22_05_15_11_14_34/0/task#bookmarks=0%2C6252&shareLine=151). Updated to also check for `.exe`.